tests/unit_tests/lapack/source/gebrd.cpp

```diff
@@ -149,6 +150,7 @@ bool usm_dependency(const sycl::device& dev, int64_t m, int64_t n, int64_t lda,
         TEST_RUN_LAPACK_CT_SELECT(
             queue, scratchpad_size = oneapi::mkl::lapack::gebrd_scratchpad_size<fp>, m, n, lda);
 #endif
+        queue.wait_and_throw();
```

Comment:
Same question about scratchpad_size

Reply:
For future reference: 
Answered here https://github.com/oneapi-src/oneMKL/pull/572#discussion_r1784569376